llvm/test/CodeGen/ARM/loopvectorize_pr33804.ll

```diff
@@ -15,16 +15,16 @@ target triple = "armv7-unknown-linux-gnueabihf"
 ; CHECK-LABEL: @cvCalcEMD2
 ; CHECK: vector.body
 ; CHECK: store <{{[0-9]+}} x ptr>
-define void @cvCalcEMD2() local_unnamed_addr #0 personality ptr @__gxx_personality_v0 {
+define void @cvCalcEMD2(ptr %dst) local_unnamed_addr #0 personality ptr @__gxx_personality_v0 {
```

Comment:
while you here, you could also drop the `local_unnamed_addr #0 personality ptr @__gxx_personality_v0`

Reply:
Fixed. Thanks!